### PR TITLE
DSG-17-Configure-LoggingContextFilter

### PR DESCRIPTION
**Pull request by afcalderonm**
## Description
add dependecy on dsgov-logging librray and implement new filter

[DSG-16](https://dol-ewf.atlassian.net/browse/DSG-16)

## Motivation and Context
it implements the logging filters needed and change the profiles name to match with the others repos

## How Has This Been Tested?
run with skaffold locally
image

## Types of changes

Patch
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation change (documentation only, will not require redeployment)
- [ ] Refactor or cleanup (functionality unchanged)
Minor
- [x] New feature (non-breaking change which adds functionality)
Major
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

- [x] I have confirmed all acceptance criteria defined by the story have been met

[DSG-16]: https://dol-ewf.atlassian.net/browse/DSG-16?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: service/build.gradle [hunk 1]
```diff
@@ -72,7 +72,7 @@ tasks.withType(JavaCompile) {
 
 jib {
 	container {
-		jvmFlags = ['-Dspring.profiles.active=deployment']
+		jvmFlags = ['-Dspring.profiles.active=local']
```
Comment (jgnuvalence) [inline service/build.gradle:75]: is this desired to be merged?

File: service/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -1,34 +0,0 @@
-spring:
```
Comment (jgnuvalence) [inline service/src/main/resources/application-local.yml:1]: just curious, is there a reason you are deleting the `application-local.ym`l file to create `application-local.yaml` ?

**Review by jgnuvalence — CHANGES_REQUESTED** [review]

File: service/build.gradle (hunk 1 above)
Comment (afcalderonm) [reply service/build.gradle:75]: yeah its was already on the main branch it's a change done by @dtsong to fix the postgres problem locally, i just renamed accordingly with the profile

File: service/src/main/resources/application-local.yml (hunk 2 above)
Comment (afcalderonm) [reply service/src/main/resources/application-local.yml:1]: yeah i tried the application-local.yml but it wasn't working and i saw @dtsong fix it creating the application-deployment.yaml, so i just renamed to application-local.yaml

**Review by jgnuvalence — APPROVED** [review]

**Pull request merged**